randovania/data/json_data/prime2.json

```diff
@@ -12946,6 +13088,84 @@
                                         }
                                     ]
                                 ],
+                                "Pickup (Darkburst)": [
```

Comment:
Add these as paths to `Door to Transit Station`.

Reply:
I'm planning to reorganize the connections here in the spirit of #103. I'll make sure to do this.